iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/errorinjection/SendMessagesErrInjTests.java

```diff
@@ -296,32 +296,16 @@ public void sendMessagesWithTcpConnectionDropNotifiesUserIfRetryExpires() throws
     private void errorInjectionTestFlowNoDisconnect(Message errorInjectionMessage, IotHubStatusCode expectedStatus, boolean noRetry) throws IOException, IotHubException, URISyntaxException, InterruptedException, ModuleClientException, GeneralSecurityException
     {
         // Arrange
-        Device targetDevice = null;
```

Comment:
Missed this in an earlier PR, but this logic is within SendMessagesCommon already, so no need to setup a new test identity here as well